### PR TITLE
Feature/295 add buttons

### LINKED ISSUE #295: Add buttons to open patient, create plan and field
I'd suggest style similar to ones below:
image

Some buttons should be inactive in certain situations (i.e. add plan should be inactive when no patient is opened).

### PR DESCRIPTION
**Pull request by ljelen**
closes #295 

Buttons:
- [x] New patient
- [x] New field
- [x] New plan

### CODE REVIEW

**Comment by grzanka** [comment]
@ljelen add some text to the buttons. Take a look as an inspiration on the github interface. All the buttons have text and colors which makes obvious what to click. Even web browsers rarely have bare icons to click on. 

Read this https://ux.stackexchange.com/questions/1795/when-to-use-icons-vs-icons-with-text-vs-just-text-links

**Comment by ljelen** [comment]
> @ljelen add some text to the buttons. Take a look as an inspiration on the github interface. All the buttons have text and colors which makes obvious what to click. Even web browsers rarely have bare icons to click on.
> 
> Read this https://ux.stackexchange.com/questions/1795/when-to-use-icons-vs-icons-with-text-vs-just-text-links

**Comment by ljelen** [comment]
> @ljelen add some text to the buttons. Take a look as an inspiration on the github interface. All the buttons have text and colors which makes obvious what to click. Even web browsers rarely have bare icons to click on.
> 
> Read this https://ux.stackexchange.com/questions/1795/when-to-use-icons-vs-icons-with-text-vs-just-text-links

image

**Comment by grzanka** [comment]
> > @ljelen add some text to the buttons. Take a look as an inspiration on the github interface. All the buttons have text and colors which makes obvious what to click. Even web browsers rarely have bare icons to click on.
> > Read this https://ux.stackexchange.com/questions/1795/when-to-use-icons-vs-icons-with-text-vs-just-text-links
> 
> image

Much better now !

**Comment by grzanka** [comment]
image


Upon starting application "Create Plan" should be inactive. It doesn't make sense to create plan when no patient is loaded. It is better to inactivate button than to display popup like below:
image

**Comment by grzanka** [comment]
@ljelen remember to fill checklist: https://github.com/pytrip/pytripgui/pull/301#issue-420241984

File: pytripgui/main_window_qt_vc/main_window_cont.py [hunk 1]
```diff
@@ -150,3 +129,20 @@ def on_about(self):
     @staticmethod
     def on_exit():
         exit()
+
+    def on_add_patinet(self):
```
Comment (grzanka) [inline pytripgui/main_window_qt_vc/main_window_cont.py:133]: typo, should `patient`, not `patinet`

**Pull request merged**